pkg/config/config_template.yaml

```diff
@@ -2365,7 +2365,7 @@ api_key:
 ## @env DD_CONTAINERD_NAMESPACE - string - optional - default: ""
 ## Activating the Containerd check also activates the CRI check, as it contains an additional subset of useful metrics.
 ## Defaults to "" which configures the agent to report metrics and events from all the containerd namespaces.
-## If you would like to limit it to watch just one namespace, specify it here.
+## To watch specific namespaces, list them separated by spaces.
 ## https://github.com/containerd/cri/blob/release/1.2/pkg/constants/constants.go#L22-L23
 #
 # containerd_namespace: k8s.io
```

Comment:
Just few notes on configuration:

- Option name should be in plural. AIUI It is still new in 7.34, so it isn't too late to rename yet.
- Normally lists in the configuration are specified as yaml lists, not space separated strings (even if this is allowed).

```suggestion.
## https://github.com/containerd/cri/blob/release/1.2/pkg/constants/constants.go#L22-L23
#
# containerd_namespaces: 
#   - k8s.io
```

WDYT?

Reply:
The `containerd_namespace` option existed in previous releases. It was a string that allowed us to select a single namespace to watch. So renaming this to `containerd_namespaces` would be a breaking change. We can consider adding an alias for it instead.

About the spaces, you are right, I was thinking about using the option via ENV, but in the config it should be a yaml list.

Reply:
For env, you should probably update the `@env` line too, for example: https://github.com/datadog/datadog-agent/blob/54edf7726d1d5842e094a6312e6f550f707fccc7/pkg/config/config_template.yaml#L1685

That, and using yaml list in the config example should be good enough.

Reply:
Fixed :+1: